dask_ml/model_selection/_incremental.py

```diff
@@ -347,3 +354,228 @@ def fit(
         scorer=scorer,
         random_state=random_state,
     )
+
+
+# ----------------------------------------------------------------------------
+# Base class for scikit-learn compatible estimators using fit
+# ----------------------------------------------------------------------------
+
+
+class BaseIncrementalSearchCV(DaskBaseSearchCV):
+    """Base class for estimators using the incremental `fit`.
+
+    Subclasses must implement the following abstract methods
+
+    * _get_ids_and_args : Build the dict of
+       ``{model_id : (param_list, additional_calls)}``
+        that are eventually passed to `fit`
+    * _get_cv_results : Build the dict containing the CV results.
+    * _get_best_estimator : Select the best estimator from the set of
+      fitted estimators
```

Comment:
I think that this should be easy to make a default for.  I would pass over all of the models in the resulting models dict and then pick the one with the best score coming out of `info`